Build Error: sf_utils.h

Hi,
when running `python setup.py develop` the build process fails. The error message is the following:
`... include/utils/sf_utils.h:29:6: error: ‘unique_ptr’ in namespace ‘std’ does not name a template type ...`

It seems there is a missing include in this file. The error can be resolved by adding 
`#include <memory>` to sf_utils.h.

--- a/nocturne/cpp/include/utils/sf_utils.h
+++ b/nocturne/cpp/include/utils/sf_utils.h
@@ -4,6 +4,8 @@
 
 #pragma once
 
+#include <memory>
+
 #include <SFML/Graphics.hpp>
 
 #include "geometry/geometry_utils.h"

